docs/page-api.md

```diff
@@ -10,7 +10,7 @@ A binding of URL paths to a Page class that can render the results for those URL
 
 * The HTTP method(s) that are acceptable.
 
-* Default: “get”
+* Default: [“get”, "head"]
```

Comment:
Oh... is it common to have an API respond to, e.g. `POST` or `HEAD`? (i.e., should we be auto-adding `head`, to everything-ish?)